common/EventLogInterceptor.js

```diff
@@ -14,7 +14,7 @@ class EventLogInterceptor {
   constructor(eventType, appType) {
     const configFileName = `eventlog-config-${appType}.yml`;
     const eventConfAbsPath = process.env.CONF_DIR ? path.join(process.env.CONF_DIR, configFileName) :
-      path.join(__dirname, '..', 'config', configFileName);
+      path.join(__dirname, 'config', configFileName);
```

Comment:
Aren't files common/config/eventlog-config-*.yml, broker specific? Shouldn't they be in broker/config? In here common/config , maybe we can have some common configs.

Reply:
`common` module is not aware of the module that is calling eventLogInterceptor.
We can make it mandatory to set CONF_DIR in testing environment but this will be an extra overhead while testing for developers (We will have to set it explicitly before running test similar to SETTINGS_PATH).
Also having CONF_DIR set to specific module configs will force testing to be done at module level. As CONF_DIR will be `broker/lib/config` for broker and `deployment-hooks/lib/config` for deployment hook module. Going forward we can make some changes in our UT framework and incorporate independent testing for each module.